stripe/resource.py

```diff
@@ -280,6 +281,36 @@ def serialize(self, previous):
 
         return params
 
+    # Override copy methods so that we can avoid having exceptions thrown as
```

Comment:
So I duplicated the comment just in case the two copy implementations become unanchored and drift apart at some future time. (i.e. another method is injected between them), and made it a precise copy so that if we want to change it it's easier to just copy and paste wholesale to the other location.

I updated the comment for better clarity though! Fix in 4bc7359.